org.osgi.test.assertj.framework/src/main/java/org/osgi/test/assertj/conditions/Mapped.java

```diff
@@ -0,0 +1,73 @@
+package org.osgi.test.assertj.conditions;
+
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+import org.assertj.core.api.Condition;
+
+public class Mapped<FROM, T> extends Condition<FROM> {
```

Comment:
This class may be replaced/changed after https://github.com/assertj/assertj-core/pull/2103

Reply:
Mapped is now part of assertj-core

https://github.com/assertj/assertj-core/pull/2193